app/src/main/res/values/strings.xml

```diff
@@ -94,40 +94,40 @@
     <string name="recovery_code_input_hint_12">Word 12</string>
     <string name="recovery_code_error_empty_word">You forgot to enter this word.</string>
     <string name="recovery_code_error_invalid_word">Wrong word.</string>
-    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all words as well as their position and try again!</string>
+    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all the words as well as their position and try again.</string>
     <string name="recovery_code_verification_ok_title">Recovery code verified</string>
-    <string name="recovery_code_verification_ok_message">Your code is correct and will work for restoring your backup.</string>
+    <string name="recovery_code_verification_ok_message">Your code is correct, so it can restore your backup.</string>
     <string name="recovery_code_verification_error_title">Incorrect recovery code</string>
     <string name="recovery_code_verification_error_message">You have entered an invalid recovery code. Please try again!\n\nIf you have lost your code, tap on \'Generate new code\' below.</string>
     <string name="recovery_code_verification_try_again">Try again</string>
     <string name="recovery_code_verification_generate_new">Generate new code</string>
     <string name="recovery_code_verification_new_dialog_title">Wait one second…</string>
-    <string name="recovery_code_verification_new_dialog_message">Generating a new code will make your existing backups inaccessible. We\'ll try to delete them if possible.\n\nAre you sure you want to do this?</string>
-    <string name="recovery_code_recreated">New recovery code has been created successfully</string>
+    <string name="recovery_code_verification_new_dialog_message">Do you want to generating a new code, making your existing backups inaccessible? They will be deleted if possible.</string>
+    <string name="recovery_code_recreated">New recovery code created</string>
     <string name="recovery_code_auth_title">Re-enter your screen lock</string>
     <string name="recovery_code_auth_description">Enter your device credentials to continue</string>
 
     <!-- Notification -->
     <string name="notification_channel_title">Backup notification</string>
-    <string name="notification_title">Backup running</string>
-    <string name="notification_backup_already_running">Backup already in progress</string>
-    <string name="notification_backup_disabled">Backup not enabled</string>
+    <string name="notification_title">Backup running…</string>
+    <string name="notification_backup_already_running">Backup already in progress…</string>
+    <string name="notification_backup_disabled">Auto-backups off</string>
 
     <string name="notification_success_title">Backup finished</string>
     <string name="notification_success_text">%1$d of %2$d apps backed up. Tap to learn more.</string>
-    <string name="notification_failed_title">Backup failed</string>
```

Comment:
There is no _text_ for this (?). Moreover, errors stopped being "Could not".

Reply:
Failed does have a specific meaning in software though, and this fits.